static/app/components/events/interfaces/crashContent/stackTrace/content.spec.tsx

```diff
@@ -172,6 +176,10 @@ describe('StackTrace', function () {
   });
 
   it('displays a toggle button when there is more than one non-inapp frame', function () {
+    MockApiClient.addMockResponse({
```

Comment:
Instead of adding this to every singe test, you can put it into a beforeEach:

```ts
describe('StackTrace', function() {
  beforeEach(() => {
    MockApiClient.addMockResponse(...)
  })
})
```

Reply:
oh beautiful, I thought it was pretty repetitive but I tried putting it after the `describe` and it wasn't working.